src/coordsim/trace_processor/trace_processor.py

```diff
@@ -11,23 +11,33 @@ class TraceProcessor():
 
     def __init__(self, params: SimulatorParams, env: Environment, trace: list,):
         self.params = params
+        self.params.inter_arr_means = []
```

Comment:
What's this for? Below you use `inter_arr_mean` not `inter_arr_means`.

Reply:
This is extra code that I forgot to delete :D Will push a commit to change it now